src/main/java/seedu/address/model/module/ModBookDate.java

```diff
@@ -66,7 +65,7 @@ public static LocalDate parseDate(String test) {
         LocalDate result = null;
         for (DateTimeFormatter f : PARSE_FORMATTERS) {
             try {
-                result = LocalDate.parse(test, f.withResolverStyle(ResolverStyle.STRICT));
+                result = LocalDate.parse(test, f);
```

Comment:
so the issue wasn't in modbookdate but in modbooktime. but since now modbookdate and modbooktime both use strict dateTimeFormatters, i just moved it to buildFormatter for better code quality or something.

Reply:
oh whoops! thank u sir ❤️